### DISCUSSION TITLE
科幻電影逐漸成真，俄烏戰爭使各國更擔憂 AI 武器

### DISCUSSION THREAD

**Discussion by yucheng1028** [1101-GS4719-C/Python-Community-GS4719-C#582] [General]
科幻電影中不乏有人工智慧反噬人類的警示內容，但似乎無法減緩近年各國發展人工智慧相關技術的趨勢，在俄烏戰爭中多次被運用的網路攻擊、深偽技術和自殺無人機等技術，使各國專家擔憂 AI 軍備競賽將成為下一次大戰的源頭。

https://technews.tw/2022/04/29/worries-regarding-ai-weapons-increased-after-russian-ukrainian-war-broke-out/

**Comment by PoooYen** [comment]
AI武器未來除非國際立法禁止，不然已經逝勢不可擋的的趨勢

**Comment by HuanYuuuuuu** [comment]
深偽技術用在諜報戰中感覺非常可怕

**Comment by cordydeng** [comment]
AI應該是幫助人們前進的動力，而不是殺人武器，如果全球能夠一起往世界和平的目標邁進，那該有多好

**Comment by Rosancl** [comment]
AI的發展是雙面刃，可以為人類的生活帶來進步與方便，也可以帶來威脅，端看人類怎麼運用。

**Comment by Lai27330** [comment]
AI的發展充滿矛盾，人們一方面希望它能自主處理複雜問題，卻又擔心AI發展出自我意識危害人類，除了技術的持續發展外也要更重視對AI的相關規範才行。

**Comment by HsuChen88** [comment]
AI終究是人寫出來的，我們應該可以在讓AI擁有自己學習的能力，也同時限制他執行的極限(像教小孩)，應該能避免AI反噬人類吧。

**Comment by HaoFish123** [comment]
感謝分享，讓我知道更多AI的用途。

**Comment by treeann** [comment]
AI的用途廣泛.....

**Comment by sys511613** [comment]
AI的發展有好有壞，如果適當使用的確可以幫助人類許多進步，希望大家能妥善利用

**Comment by stan890425** [comment]
AI的發展有好有壞，但如果沒有適當使用將會傷害人類

**Comment by jcpan821** [comment]
水能載舟，亦能覆舟

**Comment by alexthebigpp** [comment]
感覺是無可避免的，只能祈禱人類真的知道自己在做甚麼了吧

**Comment by 110409538** [comment]
科技雖然可以帶來方便,但如果用在軍事領域,會讓戰爭變得越來越可怕.

**Comment by Ben6uo** [comment]
現在只要有地方發生戰爭 大國都會進去試一下自己的新武器

**Comment by yenwei12138** [comment]
國際間真的需要快點針對AI技術擬定規範，避免以前戰爭的覆轍

**Comment by etin2468** [comment]
感覺未來再有大洲等級的大規模戰爭，就只剩下滅亡的路了

**Comment by Lynn12378** [comment]
科技發展伴隨而來的隱憂，總是會人有把科技濫用QQ

**Comment by Hchejxnc** [comment]
AI的武器的發展，若是現在沒有投入研究那未來發生戰爭時就矮別人一節，只能希望有規範能加以限制...

**Comment by Jimmy911218** [comment]
ai的用途是好是壞還是要看用的事情上而定
